imports/ui/components/modals/modals.html

```diff
@@ -15,15 +15,15 @@
 <template name="modal_earn_pti">
 
       <!-- #earnpti -->
-      <div class="main-modal-content main-modal-content-earn step-1">
+      <div class="main-modal-content main-modal-content-earn {{currentStep}}">
```

Comment:
This change is not currently being used (as the earn PTI modals are not displayed right now), but these changes allow the user to page through the instructions